lib/bloc/orders_bloc/orders_state.dart

```diff
@@ -0,0 +1,20 @@
+part of 'orders_bloc.dart';
+
+@immutable
+abstract class OrdersState {}
```

Comment:
this orders state class is very similar to other state classes. 
It is good that we have separate state for different things but why code repetition ?